### DISCUSSION TITLE
does the quickstart require a pre installed home assistant ?

### DISCUSSION THREAD

**Discussion by githubnoobjk** [dcmartin/motion-ai#22] [Q&A]
does the quickstart require a pre installed home assistant ?
Or will the quickstart install home assistant.

Does the camera have to be connected to the rpi. I don't like to take off my camera from the wall

I followed the instructions on facebook but have some errors (see my post there (johan Kruize)

**Comment by Pepeuch** [comment]
Hello, no there is no need to install home assistant first, the installation script takes care of this step as well as adding all the features that @dcmartin has previously programmed in the config file. On the other hand it will be up to you to refine all the windows and the data which are processed there (there are many), you will of course have to adapt to your configuration. ;-)

just for information the installation script works fine on jetson AGX Xavier.

**Reply by dcmartin** [reply]
Right (+1); the QuickStart documentation (https://github.com/dcmartin/motion-ai/blob/master/docs/QUICKSTART.md) should yield an operational system.  Please let me know if you have problems.  Works on Ubuntu18.04, Rasbian Buster, and Debian10 on amd64, aarch64, and arm.
